CHANGELOG.rst

```diff
@@ -1,5 +1,23 @@
 Changelog
 =========
+1.19.0 (2021-07-14)
+-------------------
+MLflow 1.19.0 includes several major features and improvements:
+
+Features:
+
+- Add support for plotting per-class feature importance computed on linear boosters in XGBoost autologging (#4523, @dbczumar)
+- Add ``mlflow_create_registered_model`` and ``mlflow_delete_registered_model`` for R to create/delete registered models.
+- Add support for setting tags while resuming run (#4497, @dbczumar)
+- Syncing with MLFlow and adding i18n support (#4490, @sunishsheth2009)
+
+Bug fixes and documentation updates:
+
+- Fix a bug causing MLflow UI to crash when sorting a column containing both `NaN` and empty values (#3409, @harupy)
+- Fix autologging event logger log_autolog_called with disable=True (#4494, @dbczumar)
```

Comment:
Can we move this entry into the `small-bug-fixes` section?

Reply:
Yes, absolutely!